projects/plugins/jetpack/extensions/blocks/payments-intro/pattern-picker.js

```diff
@@ -0,0 +1,35 @@
+import {
+	__experimentalBlockPatternSetup as BlockPatternSetup, // eslint-disable-line wpcalypso/no-unsafe-wp-apis
+} from '@wordpress/block-editor';
+import { Button, Modal } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+export default function PaymentsIntroPatternPicker( { onBlockPatternSelect, patternFilter } ) {
+	const [ isPatternSelectionModalOpen, setIsPatternSelectionModalOpen ] = useState( false );
+
```

Comment:
Let's also add another safeguard here for the experimental component.

```suggestion
	const [ isPatternSelectionModalOpen, setIsPatternSelectionModalOpen ] = useState( false );

	if ( ! BlockPatternSetup ) {
		return null;
	}
```